airbyte-integrations/connectors/source-freshcaller/source_freshcaller/manifest.yaml

```diff
@@ -118,7 +116,7 @@ definitions:
       requester:
         $ref: "#/definitions/requester"
         request_parameters:
-          has_ancestry: "true"
```

Comment:
What was this doing? ;)

Reply:
@natikgadzhi I don't think it was doing anything - I could find no reference on the freshcaller documentation